frontend/src/pages/admin/AdminEinzelnerSchueler/AdminFacharbeitApplicationForm.tsx

```diff
@@ -0,0 +1,143 @@
+/**
+ * Übersichtsseite für die Facharbeit.
+ */
+
+import React, { useState } from 'react';
+import {
+  Button, FormControl, InputLabel, MenuItem, Paper, Select, TextField,
+} from '@material-ui/core';
+import CreatePDFButton from '../../../components/Buttons/CreatePDFButton';
+import UploadedFileInformationPanel from '../../../components/FileInformationPanel/UploadedFileInformationPanel';
+
+const lehrkraefteMock = [{ id: 1, name: 'Miss X' }, { id: 2, name: 'Mister A' }, { id: 3, name: 'Lady Y' }];
+const subjectMock = [{ id: 1, name: 'Mathe' }, { id: 2, name: 'Deutsch' }, { id: 3, name: 'English' }];
+
+function AdminFacharbeitApplicationForm() {
+  const [studentName, setStudentName] = useState('');
+  const [studentClass, setStudentClass] = useState('');
+  const [choosenSubject, setChoosenSubject] = useState();
+  const [betreuendeLehrkraft, setBetreuendeLehrkraft] = useState();
+  const [unterrichtendeLehrkraft, setUnterrichtendeLehrkraft] = useState();
+  const [topicTextArea, setTopicTextArea] = useState('');
+
+  const betreuendeLehrkraefteList = lehrkraefteMock;
+  const unterrichtendeLehrkraftList = lehrkraefteMock;
+  const subjectList = subjectMock;
+
+  const handleStudentNameInputChange = (event) => {
+    setStudentName(event.target.value);
+  };
+  const handleStudentClassInputChange = (event) => {
+    setStudentClass(event.target.value);
+  };
+  const handleSubjectInputChange = (event) => {
+    setChoosenSubject(event.target.value);
+  };
+  const handleBetreuendeLehrkraftInputChange = (event) => {
+    setBetreuendeLehrkraft(event.target.value);
+  };
+  const handleUnterrichtendeLehrkraftInputChange = (event) => {
+    setUnterrichtendeLehrkraft(event.target.value);
+  };
+  const handleTopicInputChange = (event) => {
+    setTopicTextArea(event.target.value);
+  };
+  const handleSubmitFacharbeit = () => {
+    const requestBody = JSON.stringify({
+      studentId: 0,
+      topic: topicTextArea,
+      subjectId: choosenSubject,
+      responsibleTeacher: betreuendeLehrkraft,
+      teachingTeacher: unterrichtendeLehrkraft,
+    });
+    fetch('placeholder/api/facharbeit/chooseTopic', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: requestBody,
```

Comment:
der requestBody muss als JSON vorliegen, daher macht in der Zeile: body: JSON.stringify(requestBody),